pytorch/Dockerfile

```diff
@@ -109,6 +109,57 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+FROM multinode AS hf-genai
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I'd consolidate this `RUN` command and the next  one.
In general, it's encouraged to have most of `apt-get` commands to be run within one `RUN` command unless there are some technical reasons to separate them into multiple `RUN` commands.

`pip` and `conda` commands should follow the same BKM.

This is to keep the number of layers manageable and also you'll be surprised that sometimes you actually end up with much smaller containers too.

Reply:
I ended up removing these